src/solve.jl

```diff
@@ -32,60 +35,92 @@ function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algTyp
     end
 
     if typeof(alg.alg) <: OrdinaryDiffEqCompositeAlgorithm
-        interp_data = OrdinaryDiffEq.CompositeInterpolationData(integrator.sol.interp,
-                                                                interp_f)
+        ode_interp_data = OrdinaryDiffEq.CompositeInterpolationData(integrator.sol.interp,
```

Comment:
No, just ``ode_interp_data`` and ``ode_sol`` are renamed since ``dde_interp_data`` and ``dde_sol`` are added.